### PR TITLE
Fixing failing installation of ChromeDriver on Debian

### PR DESCRIPTION
**Pull request by TheNavigat**

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -14,6 +14,6 @@ else
     git rev-parse && cd "$(git rev-parse --show-toplevel)"
 fi
 
-./test/rules.sh
-./test/firefox.sh $@
+# ./test/rules.sh
+# ./test/firefox.sh $@
```
Comment (semenko) [inline test.sh]: Looks like you left these in

**Comment by semenko** [comment]
Nice catch RE chromedriver vs chromium-chromedriver!

File: test.sh (hunk 1 above)
Comment (TheNavigat) [inline test.sh]: Yea, I keep doing that. Sorry :(

**Comment by semenko** [comment]
np - ping when you want me to take a look @ pull.

**Comment by TheNavigat** [comment]
@semenko Done.

**Pull request merged**